python_modules/dagster/dagster/_core/workspace/context.py

```diff
@@ -432,6 +432,7 @@ def __init__(
         version: str = "",
         read_only: bool = False,
         grpc_server_registry=None,
+        code_server_log_level="INFO",
```

Comment:
would be nice to have all these args typed, if nothing else aesthetically so the spacing around = is the same